cookbook/specs/spec_add_union.json

```diff
@@ -0,0 +1,209 @@
+{
+    "proposal": {
+        "title": "Add Specs: Union",
+        "description": "Adding new specification support for relaying union data on Lava",
+        "specs": [
+            {
+                "index": "UNIONT",
+                "name": "union testnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDK"
```

Comment:
also import IBC

Reply:
I didn't import IBC because COSMOSSDK already imports it.

Reply:
you're right, my bad